### PR TITLE
Milestone 1 - TensorFlow ipynb File

### LINKED ISSUE #1: TensorFlow - MLP

### PR DESCRIPTION
**Pull request by ethanbenjamingabriel**
I have included the ipynb file for TensorFlow for Milestone 1

### CODE REVIEW

**Comment by ethanbenjamingabriel** [comment]
This model isn't complete as I have not included all the requirements. Please feel free to make recommendations on changes I need to make or additions I need to implement.

**Review by mushi251 — COMMENTED** [review]
Line 113: Maybe using CategoricalCrossentropy might be preferrable (based on the documentation I briefly scanned). 

Line 117: maybe fit the model separately and then use model.evaluate() on the testing set?

**Comment by trevor-yu-087** [comment]
Make sure to change learning rate, otherwise looks fine. Sparse implementations of accuracy and cross entropy are generally preferred but to be consisten with the other frameworks, probably should use the regular versions.

**Comment by trevor-yu-087** [comment]
These comments are all about the metrics, the rest of the model looks fine 
- Include the `final_training_loss` and `final_evaluation_accuracy`
- Average batch inference time should be in milliseconds, so need to multiply by a factor of 1000, since outputs of `time.time()` are in seconds
- Make the callback for `on_test_batch_begin` since we want the evaluation (inference) batch time, not the train batch time
- Just use `np.mean()` instead of writing a custom averaging function

**Review by trevor-yu-087 — APPROVED** [review]
